utils/demangle.c

```diff
@@ -1691,7 +1703,13 @@ static char *demangle_simple(char *str)
 		dd.len -= 15;
 	}
 
-	if (dd.old[0] != '_' || dd.old[1] != 'Z')
+	if (dd.old[0] != '_') {
+		if (dd.old[1] != 'Z' || dd.old[1] != 'R')
+			return xstrdup(str);
+	}
+
+	// TODO: implement demangling Rust v0 mangling
```

Comment:
Thanks for doing this.  Could you please create an issue with this?  It'd be great if it can have some examples.